packer/mongo-ami.pkr.hcl

```diff
@@ -12,7 +12,7 @@ variable "mongo_admin_url" {}
 variable "database_url" {}
 variable "okta_domain" {}
 variable "okta_api_key" {}
-variable "security_group_ids" {}
+#variable "security_group_ids" {}

```

Comment:
you dont need these?

Reply:
No, those are left over from an experiment to build the AMI in a different VPC, since the VPC we're building the AMIs in is public it doesn't need special rule sets.